docs/config/amq-broker.spec

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: amq-broker
+--- 
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: amq-broker
+  namespace: amq-broker
+  labels:
+    app: amq-broker
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: amq-broker
+  template:
+    metadata:
+      labels:
+        app: amq-broker
+    spec:
+      containers:
+      - name: amq-broker
+        image: registry.redhat.io/amq7/amq-broker:7.8
```

Comment:
Can the broker work without any volume/persistence?

Reply:
Currently, if the pod restarts, the queue-related data is lost. Do you think we need to demonstrate persistency in this doc?

Reply:
Probably we should but we can do that step wise, may be start with this version and then add persistency?

Reply:
Iterating makes sense to me.

Reply:
I created a follow-up https://issues.redhat.com/browse/USHIFT-220 issue for implementing the persistence of queues. 
It was not straightforward to reconfigure AMQ Broker to use a volume directory for its journal file.

Reply:
Can we please approve this PR if it's good in its state?

Reply:
/lgtm